src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -181,5 +181,6 @@ export default class FilingTemplate extends Vue {
 p {
   color: $gray7;
   font-size: $px-15;
+  margin-bottom: 0 !important;
```

Comment:
By default, paragraphs have 1rem bottom margin. Here, I changed them all to have 1rem top margin only, which makes it consistent and easy to handle conditional paragraphs.